### PR TITLE
[TG PORT] Removes Robot Commands and replaces it with an integrated control tablet program + silicons start with HUD on

### PR DESCRIPTION
**Pull request by itsmeow**
## About The Pull Request

~~Depends on #7338 - I will forcepush a rebased version without said commits when merged~~

Ports:
- https://github.com/tgstation/tgstation/pull/53373
- https://github.com/tgstation/tgstation/pull/53953 (fix for first PR)
- https://github.com/tgstation/tgstation/pull/54339 (fix for first PR)
- https://github.com/tgstation/tgstation/pull/56738 (fix for first PR)
- https://github.com/tgstation/tgstation/pull/61920 (fix for first PR)
- https://github.com/tgstation/tgstation/pull/64279 (fix for previous PR)
- https://github.com/tgstation/tgstation/pull/51549

yes I have planned more PRs that even depend on this PR, please send help i am in dependency hell

## Why It's Good For The Game

Borgs now have a more intuitive management system and have their own built in tablet (with limitations), it's a much better interface than verbs and also makes borgs better able to monitor the station / crew, and makes them more upgradeable via tablet apps.

There is never a case where someone really wants their HUD off, so it starts on by default now. No more roundstart button pressing!

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

image

image

image

https://user-images.githubusercontent.com/10366817/180628670-a43d82bf-3fb5-4b3e-b821-1dda56603486.mp4

</details>

## Changelog
:cl:
tweak: Borgs and AIs now start with their borgHUDs/siliconHUDs on by default.
add: Added tablets to cyborgs. Humans can install apps to it by using portable data disks from the Deluxe Silicate Selections vendor and inserting the data disk with programs uploaded to cyborgs.
add: Added a borg self-management app to integrated cyborg tablet.
del: Removed most Robot Commands verbs in favor of the app.
tweak: Some lesser-used elements on the cyborg HUD are now in the management app.
add: Cyborgs can now change their headlamp color using the integrated tablet flashlight
/:cl:

### CODE REVIEW

File: code/modules/modular_computers/file_system/programs/borg_self_monitor.dm [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+		if("printImage")
+			var/obj/item/camera/siliconcam/robot_camera/borgcam = borgo.aicamera
+			borgcam?.borgprint(usr)
+
+		if("toggleThrusters")
+			borgo.toggle_ionpulse()
+
+		if("lampIntensity")
+			borgo.lamp_intensity = params["ref"]
```
Comment (PowerfulBacon) [inline code/modules/modular_computers/file_system/programs/borg_self_monitor.dm]: Whats to stop me from setting lamp intensity to 999999999999

File: code/modules/modular_computers/file_system/programs/borg_self_monitor.dm [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+			borgo.toggle_headlamp(FALSE, TRUE)
+
+/**
+  * Forces a full update of the UI, if currently open.
+  *
+  * Forces an update that includes refreshing ui_static_data. Called by
+  * law changes and borg log additions.
+  */
+/datum/computer_file/program/borg_self_monitor/proc/force_full_update()
```
Comment (PowerfulBacon) [inline code/modules/modular_computers/file_system/programs/borg_self_monitor.dm:142]: I think theres something that does this already, either ui_update or ui_update_static_data.
Or its something on SStgui

Comment (itsmeow) [reply code/modules/modular_computers/file_system/programs/borg_self_monitor.dm:142]: There's try_update_ui which requires a UI datum, which you have to get anyway. ui_update only updates ui_data I think

Comment (itsmeow) [reply code/modules/modular_computers/file_system/programs/borg_self_monitor.dm:142]: Ah, there's a proc called `update_static_data` on datum, if the UI is null it does what this method does, but it doesn't have the tablet null check, so it won't work in this situation

**Comment by itsmeow** [comment]
technically I lied

The crew manifest is still a robot command :trollface:

**Comment by CydiaLamiales** [comment]
Would be cool to allow janitor borgs to toggle their floor auto-cleaning off and on. Just so that they don't accidentally erase any floor art or salt when there's an active revenant.

**Comment by itsmeow** [comment]
Merge #7551 first please, it will create a merge conflict that I need to fix on this PR.

**Review by PowerfulBacon — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
damn it the commits had [CI skip] in them, whoops

**Comment by itsmeow** [comment]
what

**Pull request merged**